openapi-v1.yaml

```diff
@@ -2632,6 +2632,113 @@ definitions:
       - array:read-write
       - array:admin
 
+  GroupAttributes:
+    description: Attributes describe the group itself, not any of the subgroups or assets
+    type: object
+    properties:
+      id:
+        description: the globally unique id of the group
+        type: string
+        x-omitempty: true
+      namespace:
+        description: The namespace of the group
+        type: string
+      name:
+        description: The name of the group. It is unique within the namespace. No 2 groups can have the same name
+        type: string
+      path:
+        description: A slash separated string of group names from the root to this group. Analogous to an OS filepath
+        type: string
+      description:
+        description: A human readable description of the content of the group
+        type: string
+        x-omitempty: true
+    example:
+      id: 0000-0001-0002-0003
+      namespace: my-organization
+      name: intro-to-genomics
+      path: /genomics-course/intro-to-genomics
+      description: contains arrays and notebooks for an 101 course on genomics with TileDB
+
+  GroupMember:
+    description: An asset of a group i.e an array, a udf etc but not a subgroup. These are handled and listed separately.
+    type: object
+    properties:
+      namespace:
+        description: The namespace of the asset. Groups can contain assets from multiple namespaces
+        type: string
+      name:
+        description: The name of the asset
+        type: string
+    example:
+      namespace: tiledb
+      name: median
+
+  GroupCreate:
+    description: Initial attributes for the creation of a new group
+    type: object
+    properties:
+      parent:
+        description: The name of the parent of the group. If empty, then the new group is a top level group.
+        type: string
+      name:
```

Comment:
I would expect name to not be here, rather included in path param when create

Reply:
I accept the change. This is not proper REST but is more consistent with other API functionality